docs/Analyzer Configuration.md

```diff
@@ -208,7 +208,7 @@ Option Values: Names of null check validation methods (separated by `|`) that va
 Allowed method name formats:
 
   1. Method name only (includes all methods with the name, regardless of the containing type or namespace).
-  2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format)
+  2. Fully qualified names in the [symbol's documentation ID format](https://github.com/dotnet/csharplang/blob/main/spec/documentation-comments.md#id-string-format)
```

Comment:
Is csharplang expected to move soon?

Reply:
Yep, I believe all dotnet repos are scheduled to make the switch this month, so this is just pre-emptive.